monitoring/tls/prom-alertendpoint-path-https.yaml

```diff
@@ -1,9 +1,10 @@
 prometheus:
   prometheusSpec:
     alertingEndpoints:
-      - name: v4m-alertmanager 
+      - name: v4m-alertmanager
         port: http-web
         pathPrefix: "/alertmanager"
         scheme: https
         tlsConfig:
           insecureSkipVerify: true
+    routePrefix: /prometheus
```

Comment:
This was previously being defined in the sample right? No problem with this change, just curious about the motive

Reply:
Yep!  This was originally going to be a change we made to the samples, but it made them ... a bit more confusing to use.  It would mean that, even if they did not want to expose a URL for Prometheus, they'd still have to uncomment 3 or so lines in the samples (instead of an all or nothing situation).
With this change, we can preserve the all-or-nothing approach to the samples such that, if they uncomment or change the routePrefix in the sample, the USER_DIR one would still take priority.

Reply:
Cool sounds good! Does it make sense to just remove it from the sample then? I think both TLS and Ingress samples have it